src/store/modules/accounts/actions.js

```diff
@@ -19,7 +19,7 @@ export default {
     let balance = 0
 
     try {
-      balance = await client.api.getAccountBalance(address)
+      balance = (await client.api.getAccountByPubkey(address)).balance
```

Comment:
why don't use just `await client.balance(address)`?

Reply:
It seemed easier to comprehend the change and what it is doing. Both would work I have no preference on which one stays.

Reply:
I think we should prefer higher level apis of js-sdk.

Reply:
Ok then, made the correction https://github.com/aeternity/aepp-blockchain-explorer/pull/81